templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -173,6 +173,13 @@ contents:
           connection.autoconnect-priority 100 802-3-ethernet.mtu ${iface_mtu} ${extra_phys_args}
       fi
 
+      # Move bond/vlan member interfaces to the new ovs-if-phys0 connection
+      if [[ ${iface_type} == bond || ${iface_type} == vlan ]]; then
+        new_conn=$(nmcli --get-values connection.uuid conn show ovs-if-phys0)
+        sed -i s/master=${old_conn}/master=${new_conn}/ $NM_CONN_PATH/*.nmconnection
```

Comment:
let's use nmcli con modi connection.master=${device_name}

Advantages are:
1. we don't have to modify a file and assume a path.
2. we don't need to revert the change later because using the device ID should work for both connections

Reply:
as discussed offline changed to use device name instead of UID and kept sed for simplicity